src/components/ReportTransaction.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View, Text} from 'react-native';
+import styles from '../styles/styles';
+import ReportActionPropTypes from '../pages/home/report/ReportActionPropTypes';
+import ReportActionItemSingle from '../pages/home/report/ReportActionItemSingle';
+
+const propTypes = {
+    // The chatReport which the transaction is associated with
+    // eslint-disable-next-line react/no-unused-prop-types
+    chatReportID: PropTypes.number.isRequired,
+
+    // ID for the IOU report
+    // eslint-disable-next-line react/no-unused-prop-types
+    iouReportID: PropTypes.number.isRequired,
```

Comment:
Ignoring unused prop check as these will be used in follow up PR which allows transaction rejection.